tests/image-stream/utils.spec.js

```diff
@@ -26,44 +26,34 @@ describe('ImageStream: Utils', function() {
 
   describe('.getArchiveMimeType()', function() {
 
-    it('should resolve application/x-bzip2 for a bz2 archive', function(done) {
+    it('should resolve application/x-bzip2 for a bz2 archive', function() {
```

Comment:
Oh. I'm afraid I don't know what any of that means, but does it mean that @jviotti 's original code was 'wrong'? Could this same problem also be affecting other unit-tests?